admin_requests.go

```diff
@@ -0,0 +1,41 @@
+package compute
+
+import (
+	"encoding/json"
+	"net/http"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+// EditorToken aquires and sets the editor token for the provided Runnable struct.
+// See: Runnable.Token()
+func (c *Client) EditorToken(runnable *Runnable) (*http.Response, error) {
+	// GET /api/v1/token/{environment}.{customerID}/{namespace}/{fnName}
+	req, err := c.adminRequestBuilder(http.MethodGet, path.Join("/api/v1/token", runnable.Path()), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := c.httpClient.Do(req)
+	if err != nil {
+		return res, err
+	}
+
+	token := TokenResponse{}
+	dec := json.NewDecoder(res.Body)
+	err = dec.Decode(&token)
+
+	if err != nil {
+		return res, errors.Wrapf(err, "failed to getEditorTokenFor Runnable: [%s]", runnable.Path())
+	}
+
+	if token.Token == "" {
+		return res, errors.Wrapf(errors.New("TokenReponse.Token was empty"),
```

Comment:
Do we need to check this case? I think it should be caught by the above error check 🤔

Reply:
This covers the (highly) unlikely case that the server returns a valid JSON object where the `token` field is explicitly empty. :P